wordpress-mobile/WordPress-iOS
# Issue/4322 notifications empty state

**Fixes** #4322 

**To test:**
1. Login with an account with no sites to check how the `WPNoResultsView` with an image looks.
2. Create a new site on that account and check the 5 different tabs on Notifications.
![emptysites](https://cloud.githubusercontent.com/assets/5558824/24362616/affa95cc-12e3-11e7-87ea-8b213edacbdb.png)![emptyunread](https://cloud.githubusercontent.com/assets/5558824/24362611/ae38e310-12e3-11e7-92d8-512e9b93b746.png)



 


Needs review: @jleandroperez
